### PR TITLE
Split form-elements

### LINKED ISSUE #352: Split form elements into separate files

### PR DESCRIPTION
**Pull request by aju**
closes #352

### CODE REVIEW

**Comment by mr-mig** [comment]
LGTM

File: src/docs/_data/navigation.yml [hunk 1]
```diff
@@ -33,8 +33,16 @@
...
-      location: form-elements/form-elements
+    - name: Checkbox & Radio
+      location: form-elements/checkbox-radio
+    - name: Input text
+      location: form-elements/input-text
+    - name: Selects
+      location: form-elements/select
+    - name: Textarea
+      location: form-elements/textarea
+    - name: Input text
```
Comment (kdzwinel) [inline src/docs/_data/navigation.yml]: Input text? What's that?

File: src/docs/_data/navigation.yml [hunk 2]
```diff
@@ -33,8 +33,16 @@
       location: rating/rating
     - name: Buttons
       location: buttons/buttons
-    - name: Form Elements
-      location: form-elements/form-elements
+    - name: Checkbox & Radio
+      location: form-elements/checkbox-radio
+    - name: Input text
```
Comment (kdzwinel) [inline src/docs/_data/navigation.yml]: Another input text?

**Comment by kdzwinel** [comment]
ok

**Pull request merged**